chalk-integration/src/lowering.rs

```diff
@@ -1454,6 +1469,22 @@ impl LowerGenericArg for GenericArg {
     }
 }
```

Comment:
I don't think ast needs `ConstWithoutId` struct. Parsing `ConstWithoutId` into `Const` (just like `TyWithoutId` is parsed into `Ty`) seems cleaner implementation-wise.

Reply:
Oops, I missed the right spot, I meant to comment on `impl LowerGenericArg for ConstWithoutId `